terraform/deploy/app/frontendservice.tf

```diff
@@ -0,0 +1,76 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# ECS Task Definition
+# ---------------------------------------------------------------------------------------------------------------------
+module "ecs-fargate-task-definition" {
+  source                       = "../../modules/fargate-task-definition"
+  name_prefix                  = var.name_prefix
+  container_name               = var.container_name
+  container_image              = local.container_image
+  container_port               = var.container_port
+  container_cpu                = var.container_cpu
+  container_memory             = var.container_memory
+  container_memory_reservation = var.container_memory_reservation
+  common_tags                  = local.common_tags
+  role_arn                     = "arn:aws:iam::${local.account[local.environment]}:role/${var.assume_role}"
+  account                      = lookup(local.account, local.environment)
+  log_configuration = {
+    secretOptions = []
+    logDriver     = "awslogs"
+    options = {
+      "awslogs-group"         = "/aws/ecs/${data.aws_ecs_cluster.ecs_main_cluster.cluster_name}/${var.name_prefix}"
```

Comment:
where's the corresponding log group resource?

Reply:
It may have been the one I removed.... ^^